### DISCUSSION TITLE
Reordering columns in web interface?

### DISCUSSION THREAD

**Discussion by joakimlemb** [navidrome/navidrome#2852] [Q&A] [answered]
Is it possible to reorder columns? 
The playlist view has columns as: "#, Title, Artist, etc.."
I would prefer to swap the "Title" and "Artist" places for columns so it gets displayed as "#","Artist","Title".

**Comment by deluan** [comment, marked as answer]
This is currently not possible, but will be in the rewritten UI, coming out sometime this year. We will use this React Admin feature: https://marmelab.com/react-admin/SelectColumnsButton.html

**Reply by joakimlemb** [reply]
Thanks for the quick answer, looking forward to the new UI.
